### PR TITLE
Update of tests documentation: from nose to pytest

### PR DESCRIPTION
**Pull request by GohuHQ**

### CODE REVIEW

File: doc/basic_usage.rst [hunk 1]
```diff
@@ -761,115 +760,62 @@ To address these challenges, AnyBlok ships with helper pytest fixture
...
 
-            class MyTest(SharedDataTestCase):
+Here you have an example to write a test class::
 
-               (...)
+  class TestRoom:
+    """Test Room model"""
 
-            del SharedDataTestCase
+    def test_create_room(self, rollback_registry):
```
Comment (petrus-v) [inline doc/basic_usage.rst]: I'm not sure why you are not using the fixture define above ``registry_with_blok_to_load`` ? it's a bit confusing to define something which is not used after

File: doc/basic_usage.rst [hunk 2]
```diff
@@ -761,115 +760,62 @@ To address these challenges, AnyBlok ships with helper pytest fixture
...
+  from anyblok.conftest import *  # noqa: F401,F403
 
-.. note:: it is advisable to delete the imported base class from the
-          test module, like so::
+But you can do more for example if you need a registry with one blok loaded only, we need a new fixture::
 
-            from anyblok.tests.testcase import SharedDataTestCase
+  @pytest.fixture(scope="class")
+  def registry_with_blok_to_load(request, testbloks_loaded):
+      registry = init_registry_with_bloks(['blok_to_load'], None)
```
Comment (petrus-v) [inline doc/basic_usage.rst]: I fill this example is a bit complicated for basics, blocks are installed from config file I guess

not sure where are defined request, testblocks_loaded and init_registry_with_blocks ? from ``from anyblok.conftest import *`` ?

File: doc/basic_usage.rst (hunk 1 above)
Comment (GohuHQ) [reply doc/basic_usage.rst]: Yes you are right, i'll add an example using the fixture but i want to keep this test which can be created using only the fixtures provided by AnyBlok.

Comment (petrus-v) [reply doc/basic_usage.rst]: I agree this is the most basics test to keep and probably provide in the first place

File: doc/basic_usage.rst (hunk 2 above)
Comment (GohuHQ) [reply doc/basic_usage.rst]: Ok, it makes me think that maybe it's not the right place for an example like this.
Maybe it should appear in the anyblok-book at some point, so we can provide a better explanation ?

**Comment by petrus-v** [comment]
LGTM

**Pull request merged**